modules/setup/config/__init__.py

```diff
@@ -154,11 +162,18 @@ def config_from_to(exchange, backtesting_from: int, backtesting_to: int, backtes
         backtesting_to_parsed = last_closed_candle_datetime
 
     # Check for incorrect configuration
-    if backtesting_from_ms >= backtesting_to_ms:
-        raise Exception("[ERROR] Backtesting periods are configured incorrectly.")
+    try:
+        timeframe = backtesting_from_ms - backtesting_to_ms
+        if timeframe > 0:
+            raise ConfigError()
+
+    except ConfigError:
+        ErrorOutput(sys.exc_info(),
+                    add_info="Backtesting periods are configured incorrectly.",
```

Comment:
```suggestion
                    add_info="Backtesting periods are configured incorrectly. Please make sure the 'backtesting-from' date is before the 'backtesting-to' date!",
```

I realised this error message is not really up to par - let's improve it. What do you think of this suggestion?